.gitignore

```diff
@@ -1,6 +1,9 @@
 # Bundler config
 /.bundle
 
+# local ruby env
+.ruby-version
+
```

Comment:
Should `.ruby-version` be in the gitignore? I thought we already [checked it in](https://github.com/JDTeamAcetabulum/qna/blob/master/.ruby-version). Should we remove it?